test/functionalTest/cases/0920_make_initial_notification_in_subscriptions_configurable/0920_make_initial_notification_configurable_NGSIv2.test

```diff
@@ -0,0 +1,159 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Initial Notification configurable for NGSIv2  POST /v2/entities, POST /v2/subscriptions, Dump and reset accumulator, PATCH /v2/subscriptions/subscription_id, Dump and reset accumulator.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, to create Room1 with temperature and pressure.
+# 02. POST /v2/subscriptions, to create subscription with skipInitialNotification.
+# 03. Dump and reset accumulator, see notification.
```

Comment:
Actuallly in step 03 and 05 the text should be

"Dump and reset accumulator, see no notification"

Please adjust also the `echo` statements corresponding to these steps.